Lib/idlelib/help_about.py

```diff
@@ -87,16 +87,21 @@ def create_widgets(self):
         byline.grid(row=2, column=0, sticky=W, columnspan=3, padx=10, pady=5)
 
         forums_url = "https://discuss.python.org"
-        forums = Label(frame_background, text="Python forums: "+forums_url,
-                        justify=LEFT, fg=self.fg, bg=self.bg)
-        forums.grid(row=6, column=0, sticky=W, padx=10, pady=0)
-        forums.bind("<Button-1>", lambda event: webbrowser.open(forums_url))
+        info_buttons = Frame(frame_background, bg=self.bg)
+        info_buttons.grid(row=3, column=0, columnspan=1, sticky=NSEW)
+        forums = Button(info_buttons, text='Python Discuss', width=35,
+                                 highlightbackground=self.bg,
+                                 command=lambda: webbrowser.open(forums_url))
+        forums.grid(row=3, column=0, sticky=W, padx=10, pady=10)
+
+
         docs_url = ("https://docs.python.org/%d.%d/library/idle.html" %
                     sys.version_info[:2])
-        docs = Label(frame_background, text=docs_url,
-                     justify=LEFT, fg=self.fg, bg=self.bg)
-        docs.grid(row=7, column=0, columnspan=2, sticky=W, padx=10, pady=0)
-        docs.bind("<Button-1>", lambda event: webbrowser.open(docs_url))
+        docs = Button(info_buttons, text='Python Documentation', width=35,
```

Comment:
This URL is specifically for IDLE.  This is redundant with the IDLE Docs entry in the HELP menu, but this PR is about button instead of label.

```suggestion
        docs = Button(info_buttons, text='IDLE Documentation', width=35,
```